### DISCUSSION TITLE
Sometimes difficulty star rating is shown as 0 stars

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#15694] [Q&A] [answered]
Seemingly at random, difficulty for a beatmap is shown as 0 stars. Sometimes there are maps that are 0* normally, and show correct star rating when any mod is applied, and sometimes the opposite is true. I haven't been able to confirm whether it happens only when you are logged out, because it's really difficult to reproduce this issue (requires manually going through the difficulties for all beatmaps in your library). Relaunching the game just makes a different beatmap have this issue. Sorry for the lack of details.
In this clip, I found a rare case where 3 difficulties on the same beatmapset are affected by the issue. 2 of the difficulties show correct star rating only with a mod enabled, and one difficulty which is only correct without the mod:

https://user-images.githubusercontent.com/53346335/142723843-40fb4d7a-4286-4ac7-9573-01df700fb27d.mp4

Logs:
[runtime.log](https://github.com/ppy/osu/files/7574411/runtime.log)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15699
